src/main/java/juja/microservices/users/Users.java

```diff
@@ -40,6 +43,11 @@ public MethodValidationPostProcessor methodValidationPostProcessor() {
         return new MethodValidationPostProcessor();
     }
 
+    @Configuration
+    @EnableScheduling
+    public static class SchedulingConfiguration {
```

Comment:
Why did we use this nested configuration class?
Can we apply @EnableScheduling to main configuration class?

Reply:
@VadimDyachenko 
please pay attention on comment